dhis-2/dhis-api/src/main/java/org/hisp/dhis/deduplication/MergeObject.java

```diff
@@ -43,4 +43,6 @@
     private List<String> trackedEntityAttributes;
 
     private List<String> relationships;
+
+    private List<String> enrollments;
```

Comment:
We will need to init this and add @Builder.default when #8596 is merged.

Reply:
And @JsonProperty :)

Reply:
Done